src/main/java/io/appform/campaignmanager/resources/Root.java

```diff
@@ -261,4 +281,26 @@ public Response downloadItemsSnapshot(
                                      .get("senders"));
     }
 
+
+    /**
+     * Resolves the specified template text, with replacement parameters specified in the specified {@link CSVRecord}.
+     *
+     * @param record the replacement parameters embedded in the {@link CSVRecord} instance
+     * @param templateText the template text that needs to be resolved
+     *
+     * @return the resolved text
+     */
+    private StoredSmsNotificationItem getNotificationItem(CSVRecord record, String templateText) {
+
+       Map<String, String> map = record.toMap();
```

Comment:
Replace with val

Reply:
just trying to understand here - whats the benefit of using val instead of using the actual type?